Mathlib/Algebra/Quaternion.lean

```diff
@@ -102,6 +102,10 @@ theorem mk.eta {R : Type*} {c₁ c₂} (a : ℍ[R,c₁,c₂]) : mk a.1 a.2 a.3 a
 
 variable {S T R : Type*} [CommRing R] {c₁ c₂ : R} (r x y z : R) (a b c : ℍ[R,c₁,c₂])
 
+instance : Nonempty ℍ[R, c₁, c₂] := ⟨⟨c₁,c₁,c₁,c₁⟩⟩
```

Comment:
```suggestion
instance : Inhabited ℍ[R, c₁, c₂] := ⟨0⟩
```
would be more canonical

Reply:
I think TC is fast enough when it comes to finding a zero element.

Reply:
this Inhabited instance already exists further down the file, L189. Should I just delete this?

Reply:
Deleting it sounds good to me.